doc/modules/multimap.md

```diff
@@ -353,6 +354,18 @@ SPAMHAUS_PBL_BLACKLIST {
 }
 ~~~
 
+## Extract from
```

Comment:
This should be moved to the attributes section, not worth a special section for this specific attribute.